scripts/core/common.yml

```diff
@@ -249,6 +249,12 @@ etors:
       desc: "Invalid host pointer"
     - name: ERROR_INVALID_USM_SIZE
       desc: "Invalid USM size"
+    - name: ERROR_INVALID_COMMAND_BUFFER_EXP
+      desc: "Invalid Command-Buffer"
+    - name: ERROR_INVALID_COMMAND_BUFFER_SYNC_POINT_EXP
+      desc: "Sync point is not valid for the command-buffer"
+    - name: ERROR_INVALID_COMMAND_BUFFER_SYNC_POINT_WAIT_LIST_EXP
+      desc: "Sync point wait list is invalid"
```

Comment:
I feel like these should be towards the end of the `etors` list, with only `ERROR_UNKNOWN` below. I also think they should have explicit values, starting at `0x1000` to separate them from the range of core spec results.

Reply:
Good idea, I've made this change now.